client/blocks/eligibility-warnings/index.jsx

```diff
@@ -57,14 +72,19 @@ function getHoldMessages( translate ) {
 			description: translate( 'You must be the owner of the primary domain subscription to use this feature.' ),
 			supportUrl: 'https://support.wordpress.com/'
 		},
-		SITE_GRAYLISTED: {
-			title: translate( 'Flagged site' ),
-			description: translate( 'This feature is not supported on sites that are not in good standing.' ),
+		NO_WPCOM_NAMESERVERS: {
+			title: translate( 'No WordPress.com name servers' ),
+			description: translate( 'Your custom domain must point to WordPress.com name servers.' ),
 			supportUrl: 'https://support.wordpress.com/'
 		},
-		SITE_PRIVATE: {
-			title: translate( 'Private site' ),
-			description: translate( 'This feature is not supported on private sites.' ),
+		NOT_RESOLVING_TO_WPCOM: {
+			title: translate( 'Primary domain not pointing to WordPress.com servers' ),
+			description: translate( 'Your primary domain must point to WordPress.com name servers.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Your custom domain must point to WordPress.com name servers.' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/eligibility-holds-messages)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).